tests/unit/mock/config/compliance/compliance/hp_comware/hp_comware_backup.txt

```diff
@@ -0,0 +1,48 @@
+sysname HP-JKT-01
+#
+dhcp enable
+dhcp server forbidden-ip 10.10.10.1 10.10.10.100
+dhcp server always-broadcast
+#
+dhcp server ip-pool CKP
+ gateway-list 10.10.10.1
+ domain-name intra.data.co.id
+ expired day 30
+ netbios-type b-node
+#
+bgp 65330
+ router-id 10.10.10.254
+ graceful-restart
+ graceful-restart timer restart 120
+ graceful-restart timer wait-for-rib 360
+ peer 10.20.240.1 description ***Point to Point Connection**
+ peer 10.20.240.1 ebgp-max-hop 10
+ peer 10.30.240.1 password cipher $x$x$xxxxxxxxxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxx==
+  address-family ipv4 unicast
+  balance 4
+  peer 10.30.240.1 enable
+  peer 10.30.240.1 route-policy P2P-FirstTry import
+  peer 10.30.240.1 route-policy P2P-FirstTry export
+#
+snmp-agent
+snmp-agent local-engineid 8000XXX123456789AB503C00000001
+snmp-agent community read RO
+snmp-agent community read read
+snmp-agent community read ro
+#
+acl advanced name HPE
+ rule 1 permit source 10.180.50.254 0 destination 10.1.0.249 0
+ rule 2 permit source 10.180.50.0 0.0.0.127 destination 10.1.4.62 0
+ rule 3 permit source 10.180.50.0 0.0.0.127 destination 10.2.4.62 0
+#
+header motd #
```

Comment:
Can you add 
* a one line motd
* A non-compliant banner

Reply:
done

test: ✅ Add non-compliant banner test

Reply:
sometimes 1 banner line is like

```
header motd "my banner"
```

Which doesn't have the same delimiters, trying to verify that as well.

Reply:
Hi @itdependsnetworks ,

I want to make sure about CiscoConfigParser banner parser.
I saw that the delimiter was set statically: `_, delimeter, banner = config_line.partition("^C")`

Is the code supported whenever the banner is delimited using `$` or `#` instead of `^C`.

Because, I have difficulties to dynamically get HP delimiter, and want to set the partition parameter with `#`.
So I can support the single/multiline banner with `#` delimiter only. (like Cisco using ^C)

Reply:
nvm :)
i create a revert caret function to return the dynamic delimiter

Reply:
Hi @itdependsnetworks,
Instead of handling multiline banner, we also finished handle the singleline banner as your suggest above.

I wanted to let you know that I handle two types of delimiter:
1. caret_c `^C` - for example: 
`header motd ^CThis is a single line banner^C`
2. Single delimiter - for example: 
`header motd "This is a single line banner"`, or 
`header motd #This is a single line banner#`

My colleague @bintangf will be responsible for creating the unit testing, as well as the MR. 
Please let us know if you have any additional input other than the single-line banner above.

Best regards,

Reply:
I have added more commits about single-line banners and banners with different delimiters.